scripts/sign_encrypt.py

```diff
@@ -125,6 +132,7 @@ def main():
     from Crypto.Hash import SHA256
     from Crypto.PublicKey import RSA
     from Crypto.Util.number import ceil_div
+    from Cryptodome.Cipher import AES
```

Comment:
Why is Cryptodome used instead of Crypto?

Reply:
OK, I'm just concerned that we're adding dependencies that could be avoided.
So now we're using both PyCrypto and PyCryptodome.
Perhaps we should switch to use PyCryptodome only, out of scope for this PR of course.

Reply:
> Perhaps we should switch to use PyCryptodome only, out of scope for this PR of course.

Sounds good to me.